internal/credit/owner_connector.go

```diff
@@ -0,0 +1,32 @@
+package credit
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/openmeterio/openmeter/internal/streaming"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type OwnerConnector interface {
```

Comment:
who is the "owner"?

Reply:
whoever owns the grant, currently only entitlements. the idea was that in the future if we want e.g. feature specific grants, or a general credit system, we won't have to change a thing just reimplement this interface.